ipc/provider/src/manager/evm/manager.rs

```diff
@@ -778,9 +783,123 @@ impl EthSubnetManager {
     }
 }
 
+#[async_trait]
+impl BottomUpCheckpointRelayer for EthSubnetManager {
+    async fn submit_checkpoint(
```

Comment:
Should we make these checkpoint-related methods available through the `IpcProvider`?

Reply:
I donno if that's required. But we can easily add them when we have to. I would say we keep it as it is, then expose them when needed to.

Reply:
Resolved in comment above.